core/tcp_connection.h

```diff
@@ -3,20 +3,42 @@
 #include "dronecore.h"
 #include "dronecore_impl.h"
 #include "connection.h"
+#include<sys/types.h>
```

Comment:
nitpick: space after `#include`.

Reply:
@julianoes Shouldn't `make fix_style` catch this sort of thing?